scripts/core/editor3/components/Editor3.jsx

```diff
@@ -85,7 +85,7 @@ export class Editor3Component extends React.Component {
             'text/html',
             'Files'
         ].indexOf(mediaType) !== -1;
-        const supportsImages = !readOnly && !singleLine && editorFormat.indexOf('picture') !== -1;
+        const supportsImages = !readOnly && !singleLine && editorFormat.indexOf('media') !== -1;
```

Comment:
You've missed one here `const supportsMedia`, make sure you replace where it is used too.